src/wayland_hook.h

```diff
@@ -0,0 +1,2 @@
+extern void* wl_handle;
+extern void* wl_display_ptr;
```

Comment:
There are clients which connect to the same display multiple times. Does this need to consider that?

Reply:
fwiw it might be good to look at how a similar situation is handled on X11, if possible?

Reply:
> There are clients which connect to the same display multiple times. Does this need to consider that?

not sure but it seems to work with everything that I have tried so far